samples/get-instance-config.js

```diff
@@ -15,7 +15,7 @@
 
 // sample-metadata:
 //  title: Gets the instance config metadata for the configuration nam6
-//  usage: node get-instance-config.js <PROJECT_ID> <INSTANCE_CONFIG_ID>
+//  usage: node get-instance-config.js <PROJECT_ID>
```

Comment:
Why are we not sending `INSTANCE_CONFIG_ID` ?

Reply:
No, in the old sample we are not seding the parameter. We are just getting the NAM6 configurations